src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Forms/form_seo.html.twig

```diff
@@ -26,6 +26,9 @@
 
   <h2>{{ 'Search Engine Optimization'|trans({}, 'Admin.Catalog.Feature') }}</h2>
   <p class="subtitle">{{ 'Improve your ranking and how your product page will appear in search engines results.'|trans({}, 'Admin.Catalog.Feature') }}</p>
+  
+  <p>{{ "Search engine result preview:"|trans({}, 'Admin.Catalog.Feature') }}</p>
```

Comment:
Wouldn't it be nice to have a more user-friendly phrase here? Like "Get a sneak peek at it:", "Have a look at the result:", or "Here is a preview of your search engine result, play with it!".

Reply:
I like the last one `Here is a preview of your search engine result, play with it!` :+1:

Reply:
Indeed, I requested a better phrase on Jira, the current one needs to be replaced. :)

ping @colinegin

Reply:
Love it !